marklogic-data-hub/src/test/java/com/marklogic/hub/HubTestBase.java

```diff
@@ -158,21 +162,6 @@
     @Autowired
     protected Versions versions;
 
-    @Autowired
```

Comment:
Got rid of these, as it was hurting my head trying to figure out whether modifying one of these would affect future tests. There's no need for them to be Spring beans, it's easier to think about if we just instantiate a command when we need one.